src/serializer.ts

```diff
@@ -24,6 +24,15 @@ interface TransactionJSON {
   TxnSignature?: string
 }
 
+interface PaymentTransactionJSONAddition extends PaymentJSON {
+  TransactionType: 'Payment'
```

Comment:
I think you should include the amount and destination field here?

Reply:
it is included, by virtue of extending `paymentJSON` :)